### PR TITLE
[doc] network_mode can also take the name of a network as value.

### PR DESCRIPTION
**Pull request by ticosax**

### CODE REVIEW

File: docs/hostconfig.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ of docker remote API. Otherwise they are ignored.
 container ('bridge': creates a new network stack for the container on the
 Docker bridge, 'none': no networking for this container, 'container:[name|id]':
 reuses another container network stack), 'host': use the host network stack
```
Comment (blueyed) [inline docs/hostconfig.md]: The `)` should be at the end?!

File: docs/hostconfig.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ of docker remote API. Otherwise they are ignored.
...
-inside the container.
+inside the container or any name that identifies an existing network.
```
Comment (blueyed) [inline docs/hostconfig.md]: s/an existing network/an existing Docker network/ ?

**Comment by ticosax** [comment]
thanks @blueyed I took your comment into consideration and pushed a new version.

**Comment by dnephin** [comment]
LGTM

**Pull request merged**